products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
FWIW at least in the local docker environment it was working, at least the way I reasoned it.
* Awestruct made the substitution which was assigned into an AsciiDoctor Attribute.
* When AsciiDoctor substituted the attribute in the included flle later, it was using the attribute value which had already been set from the main file, 
* This bypassing the need for awestruct.to interpret it in the _included file which awestruct didn't process.

The only thing that was weird, which I believe I figured out has nothing to do with what I did is that the image would get document title text included in it from someplace.